components/cache/adapters/chain_adapter.rst

```diff
@@ -0,0 +1,26 @@
+.. index::
+    single: Cache Pool
+    single: Chain Cache
+
+Chain Cache Adapter
+===================
+
+This adapter allows to combine any number of the previous adapters. Cache items
+are fetched from the first adapter which contains them. Besides, cache items are
+saved in all the given adapters, so this is a simple way of creating a cache
+replication::
```

Comment:
Using `so this is a simple way of creating a layered cache, from fastest to slowest storage engines::`